draft-ietf-cbor-packed.md

```diff
@@ -817,11 +864,13 @@ this requires additional consideration.)
 Examples
 ========
 
-The (JSON-compatible) CBOR data structure depicted in
-{{fig-example-in}}, 400 bytes of binary CBOR, could lead to a packed
-CBOR data item depicted in {{fig-example-out}}, ~309 bytes.  Note that
-this particular example does not lend itself to prefix compression, so
-it uses the simple common-table setup form (tag 113).
+The (JSON-compatible) CBOR data structure depicted in {{fig-example-in}},
+400 bytes of binary CBOR, could be packed into the CBOR data item depicted
+in {{fig-example-out}}, ~309 bytes, only employing item sharing.
```

Comment:
Why is this given as an approximate of 309 bytes? cbor.me actually gives me 308 bytes, see [here](https://cbor.me/?diag=113([[%22price%22,%20%22category%22,%20%22author%22,%20%22title%22,%20%22fiction%22,%208.95,%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%22isbn%22],%20%20%20%20/%20%200%20%20%20%20%20%20%20%20%20%201%20%20%20%20%20%20%20%20%202%20%20%20%20%20%20%20%20%203%20%20%20%20%20%20%20%20%204%20%20%20%20%20%20%20%20%205%20%20%20%206%20%20%20/%20%20%20%20%20{%22store%22:%20{%20%20%20%20%20%20%20%22book%22:%20[%20%20%20%20%20%20%20%20%20{simple(1):%20%22reference%22,%20simple(2):%20%22Nigel%20Rees%22,%20%20%20%20%20%20%20%20%20%20simple(3):%20%22Sayings%20of%20the%20Century%22,%20simple(0):%20simple(5)},%20%20%20%20%20%20%20%20%20{simple(1):%20simple(4),%20simple(2):%20%22Evelyn%20Waugh%22,%20%20%20%20%20%20%20%20%20%20simple(3):%20%22Sword%20of%20Honour%22,%20simple(0):%2012.99},%20%20%20%20%20%20%20%20%20{simple(1):%20simple(4),%20simple(2):%20%22Herman%20Melville%22,%20%20%20%20%20%20%20%20%20%20simple(3):%20%22Moby%20Dick%22,%20simple(6):%20%220-553-21311-3%22,%20%20%20%20%20%20%20%20%20%20simple(0):%20simple(5)},%20%20%20%20%20%20%20%20%20{simple(1):%20simple(4),%20simple(2):%20%22J.%20R.%20R.%20Tolkien%22,%20%20%20%20%20%20%20%20%20%20simple(3):%20%22The%20Lord%20of%20the%20Rings%22,%20%20%20%20%20%20%20%20%20%20simple(6):%20%220-395-19395-8%22,%20simple(0):%2022.99}],%20%20%20%20%20%20%20%22bicycle%22:%20{%22color%22:%20%22red%22,%20simple(0):%2019.95}}}])).

Reply:
I think we need to go through all these numbers after we have whittled down all the small fixes.
Now https://github.com/cbor-wg/cbor-packed/issues/15